README.md

```diff
@@ -395,7 +395,8 @@ supported by this (and past) versions of this repo (go-github).
 Versions prior to 48.2.0 are not listed.
 
 | go-github Version | GitHub v3 API Version |
-| ----------------- | --------------------- |
+|-------------------| --------------------- |
+| 53.3.0            | 2022-11-28            |
```

Comment:
Please revert these changes, as they are controlled by maintainers of this repo only.

Reply:
Thanks for the info, reverted